### PR TITLE
[RFC] sipeed_tang_mega_138k: new board

### PR DESCRIPTION
**Pull request by Icenowy**
Tagged RFC now because the board (and even the chip on my board) is still at ES stage.

### CODE REVIEW

File: litex_boards/platforms/sipeed_tang_mega_138k.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        IOStandard("LVCMOS33")
+    ),
+
+    # Leds
+    ("led_n", 0,  Pins("J14"), IOStandard("LVCMOS33")),
+    ("led_n", 1,  Pins("R26"), IOStandard("LVCMOS33")),
+    ("led_n", 2,  Pins("L20"), IOStandard("LVCMOS33")),
+    ("led_n", 3,  Pins("M25"), IOStandard("LVCMOS33")),
+    ("led_n", 4,  Pins("N21"), IOStandard("LVCMOS33")),
+    ("led_n", 5,  Pins("N23"), IOStandard("LVCMOS33")),
```
[the remark this answers is not in the document]
Comment (Icenowy) [reply litex_boards/platforms/sipeed_tang_mega_138k.py:35]: Well at least I use _n to mean it's active low.

[the remark this answers is not in the document]
Comment (josuah) [reply litex_boards/platforms/sipeed_tang_mega_138k.py:35]: > Well at least I use _n to mean it's active low.

This makes more sense to me that way too. Thank you for clarifying!

What a carefully crafted board! Instructive for myself...

**Comment by enjoy-digital** [comment]
@Icenowy: We can probably merge now that the board seems to be available. Thanks a lot!

**Comment by Icenowy** [comment]
Well this PR targets ES board with ES chip. Maybe further change will be needed for final product board, I wonder whether Sipeed people want to do this.

**Pull request merged**